reactor-core/src/main/java/reactor/core/publisher/FluxTimeout.java

```diff
@@ -134,11 +122,12 @@ static String addNameToTimeoutDescription(Publisher<?> source,
 		TimeoutMainSubscriber(CoreSubscriber<? super T> actual,
 				Function<? super T, ? extends Publisher<V>> itemTimeout,
 				@Nullable Publisher<? extends T> other,
-				@Nullable String timeoutDescription) {
-			super(actual);
+				@Nullable String timeoutDescription, Publisher<?> firstTimeout) {
```

Comment:
let's order things logically, put `firstTimeout` before `itemTimeout` (and same in fields order above)